docs/Getting Started.md

```diff
@@ -1,39 +1,63 @@
 # Getting Started with Inductiva
-Welcome to Inductiva, a Python API package that enables running popular [open-source physical system simulation packages]() in the Cloud. Experience HPC-standard scalability, optimized performance, and cost-efficiency with minimal setup, all through Inductiva's streamlined interface.
+Welcome to Inductiva, a Python API package that enables running popular [open-source physical system simulation packages]() 
+in the Cloud. 
 
-### What We'll Cover:
+This tutorial provides a step-by-step guide to help you get started with Inductiva 
+before running your first simulation project. Should you encounter any errors or 
+issues along the way, our [troubleshooting guide]() is readily available to assist 
+you in resolving them efficiently.
 
-* [Installing Inductiva API]()
-* [What to read next]()
+### Steps We'll Cover:
 
+1. [Request Your API Access Token]()
+2. [Install the Inductiva API]()
```

Comment:
I would do reverse the order. 1 <-> 2

Let's assume that the person has already requested the key, so it's just a matter of configuring it. 
We will still leave the link to request the key if you don't have it. But I think if we have the key request first, we will loose people because our onboarding is still pretty bad... :(

And actually, we will be "sending" keys to people for now...